MOVIShield.cpp

```diff
@@ -45,11 +45,13 @@
 
 MOVI::MOVI()
 {
+    usehardwareserial=false;
     construct(ARDUINO_RX_PIN, ARDUINO_TX_PIN, false);
 }
 
-MOVI::MOVI(bool debugonoff)
+MOVI::MOVI(uint8_t debugonoff)
```

Comment:
...while you are at it. Otherwise, I'll just do it next time I am working on the library.

In anyways: Thank you very much for your contribution!